lib/utils/urlUtils.js

```diff
@@ -41,7 +41,33 @@ function getAllButProtocolAndHost(url) {
   return rest;
 }
 
+/**
+ * Decodes the url from an xml compliant string uri
+ * @param {string} url the url
+ * @returns {string} the decoded url
+ */
+function decodeFromXML(url) {
+  if (url === undefined || url === null || url === '') {
+    return '';
+  }
+
+  const regexAmpersand = new RegExp('&amp;', 'g'),
```

Comment:
@LuisTejedaS Is this list of ampersand characters exhaustive? i.e. can there be other characters as well that are not included here?